eo-maven-plugin/src/test/java/org/eolang/maven/ParseMojoTest.java

```diff
@@ -84,22 +84,22 @@ void failsOnTimeout(@TempDir final Path temp) {
     @Test
     @ExtendWith(OnlineCondition.class)
     void parsesWithCache(@TempDir final Path temp) throws Exception {
-        final FakeMaven maven = new FakeMaven(temp);
         final Path cache = temp.resolve("cache");
+        final FakeMaven maven = new FakeMaven(temp)
+            .withProgram("invalid content")
+            .with("cache", cache);
         final String expected = new UncheckedText(
             new TextOf(new ResourceOf("org/eolang/maven/main.xmir"))
         ).asString();
-        final String hash = new ChNarrow(new ChRemote("0.25.0")).value();
+        final CommitHash hash = new ChNarrow(new ChRemote("0.25.0"));
```

Comment:
@volodya-lombrozo since `hash` is used more than one time I would decorate it with `ChCached`

Reply:
@maxonfjvipon Added. Thanks.